appinventor/appengine/src/com/google/appinventor/server/storage/ObjectifyStorageIo.java

```diff
@@ -1780,6 +1802,25 @@ public void run(Objectify datastore) {
                 }
                 recovered = true;
                 result.t = resultBuffer.array();
+                // Should we downgrade to the blobstore (for debugging)?
+                // Note: We only run if we have at least 5 seconds of runtime left in the request
+                long timeRemaining = ApiProxy.getCurrentEnvironment().getRemainingMillis();
+                if (conversionEnabled && !useGcs &&
+                  (timeRemaining > 5000)) {
```

Comment:
Same here. I think it's easier to read if it's all on one line:
    if (conversionEnabled && !useGcs && (timeRemaining > 5000)) {
But, this is optional. If you prefer it on two lines, that's fine.